Signal/src/ProfileFetcherJob.swift

```diff
@@ -40,16 +40,20 @@ class ProfileFetcherJob: NSObject {
 
     public func getProfile(recipientId: String, remainingRetries: Int = 3) {
 
-        if let lastDate = ProfileFetcherJob.fetchDateMap[recipientId] {
-            let lastTimeInterval = fabs(lastDate.timeIntervalSinceNow)
-            // Don't check a profile more often than every N minutes.
-            let kGetProfileMaxFrequencySeconds = 60.0 * 5.0
-            if lastTimeInterval < kGetProfileMaxFrequencySeconds {
-                Logger.info("\(self.TAG) skipping getProfile: \(recipientId), lastTimeInterval: \(lastTimeInterval)")
-                return
+        // Only throttle profile fetch in production builds in order to
+        // facilitate debugging.
+        if !_isDebugAssertConfiguration() {
```

Comment:
The larger the code change between debug vs. release the more nervous I get. What about:

```
let kGetProfileMaxFrequencySeconds = _isDebugAssertConfiguration()  ? 0 :  60.0 * 5.0
```